Xamarin.PropertyEditing.Mac/Controls/UnfocusableTextView.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using AppKit;
+using CoreGraphics;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	class UnfocusableTextView : NSTextView
```

Comment:
NS-prefix is NeXtStep. I don't think we should imply out control is Native with the NS-prefix. With code completion so prevalent I don't think having long descriptive class names is a major issue for developers.